metatron-app/metatron-discovery
# fix metadata detail and explore data main error

### Description

Fix metadata detail, exploration data error
1.  fix navigate from another screen to explore error
2. fix favorite list in explore main error
3. fix columns tab in metadata detail modal css
4. fix explore main new/update, favorite list popularity error
5. fix explore main popular api request size 
6. fix metadata detail header text error

**Related Issue** : 



### How Has This Been Tested?

1. check error is occured when navigate from some screen to exploretaion data
2. check favorite list in main screen shows properly when toggle metadata favorite
3. check columns tab's popularity bar position in metadata detail modal css - 
4. check new/update, favorite list popularity shows properly
5. check popular api request is proper
6. check metadata detail header text in information and column tab properly only show metadata type is datasource
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
